pkg/util/util.go

```diff
@@ -210,12 +211,12 @@ func BreakingChangeAllowed(left string, right string) (bool, error) {
 	return !minor, err
 }
 
-func PrintDelimiterLine(delimiterChar string) {
+func PrintDelimiterLine(w io.Writer, delimiterChar string) {
```

Comment:
This signature changing breaks the API from a semver perspective. If this were the Helm client we would have kept the existing function, deprecated it, created a new function doing what we want, and then used that all over.

Reply:
chart-testing was never meant to be used as API and it is not advertized as such. Admittedly, we don't hide the code in an internal package. I'm gonna hold back merging this until we get consensus on how to proceed.

Reply:
OK, so I went ahead and added a new function so we can get this out soon. We should still have the discussion on how we want to generally handle this.

Reply:
I think this makes sense 👍  We don't explicitly advertise exported semver compliance for chart-testing, but I think it may be implied 🤔  For now the new function seems like a safe approach to fix quickly without making a breaking decision. I'm making a review note about adding a deprecation comment.